api/server.js

```diff
@@ -37,3 +40,49 @@ const port = APP_CONFIG.PORT || 3000;
 app.listen(port, () =>
   console.log(`Server started on http://localhost:${port}`)
 );
+
+let runMonitor = false;
+
+const monitor = () => {
+  if (runMonitor) {
+    console.log(`[${dayjs().utc()}] Running monitor`);
```

Comment:
As per our convo, should this be...
```suggestion
    console.log(`[${dayjs.utc()}] Running monitor`);
```

Or are you wanting this to be the "local" time?